README.md

```diff
@@ -0,0 +1,16 @@
+# LLVM Build Dashboard
+
+## Project Description
+A lightweight dashboard designed to display the build status of LLVM Build Bots in an easily interpretable fashion via a User Interface.
```

Comment:
nit: This is personal preference, but I like [line wrapping documentation](https://about.gitlab.com/blog/2016/10/11/wrapping-text/) (consider 80 character line wraps).

Reply:
Lines wrapper here 284f850

Reply:
ditto

`A lightweight dashboard designed to display the build status of LLVM Build Bots in an easily interpretable` has 107 characters, which does not adhere to any line wrap convention

Reply:
Updated here a3cd071